rpc/api.proto

```diff
@@ -24,6 +24,8 @@ service WalletService {
 	rpc Accounts (AccountsRequest) returns (AccountsResponse);
 	rpc Balance (BalanceRequest) returns (BalanceResponse);
 	rpc GetTransactions (GetTransactionsRequest) returns (GetTransactionsResponse);
+	rpc TicketPrice(TicketPriceRequest) returns (TicketPriceResponse);
```

Comment:
Please add the extra space between the method name and the parameter type for consistency